packages/examples/cvat/recording-oracle/run.py

```diff
@@ -1,9 +1,29 @@
 import uvicorn
+from human_protocol_sdk.constants import KVStoreKeys
+from human_protocol_sdk.kvstore import KVStoreClient
 
+from src.chain.web3 import get_web3
 from src.core.config import Config
 
+
+def set_public_key_to_kvstore() -> None:
+    if Config.encryption_config.pgp_public_key_url:
+        w3 = get_web3(Config.polygon_amoy.chain_id)
+        kvstore_client = KVStoreClient(w3)
+
+        if not kvstore_client.get_public_key(Config.polygon_amoy.addr):
```

Comment:
Probably, these should not have amoy hardcoded.

Reply:
Yes, definitely, I've fixed that